faq.html

```diff
@@ -0,0 +1,54 @@
+<body class="page-home">
+  <section>
+      <h1>FAQs - summer project</h1>
+      <br>
+      <div align="left">
+          <p>Q: What is this programme about?<p>
+          <p>A: We at ILUGD have thought of involving people who can dedicate time during this summer to open source projects so that it can help them hone their skills.
+        We are still in the planning phase of what projects will be in for this summer , but they will be announced very soon. This is a great opportunity for newbies starting out in open-source to get involved and collaborate with amazing people !</p>
+        
+        <p>Q: Which communities are monitoring these projects?</p>
+        <p>A: Indian Linux Users Group , Delhi(ILUGD) is taking incharge of it.</p>
+        
+        <p>Q: What is the time duration of the project?</p>
+        <p>A: Each project is around two-month long.</p>
+        
+        <p>Q: What is the eligibility criteria?</p>
+        <p>A: There's no eligibility criteria . Everyone is welcome to contribute . However please fill out <a href = "https://docs.google.com/forms/d/1PrGlfqy6kB2Ap_2j2vu91U0aLsybBI1cZB8umqlY5Ok/edit" style="color:red">this</a> google form if you ish to participate.
+        
+        <p>Q: How much time do I need to dedicate to the project?</p>
+        <p>A: There's no hard and fast rule for committing a bare minimum number of hours . <br>You can fix up the number of hours per week according to what suits you .</p>
+        
+        <p>Q: Can I be a mentor for this programme?</p>
+        <p>A: Definitely , just select the role as mentor in the google doc and we will set you up.</p>
+        
+        <p>Q: Is there a stipend for the programme?</p>
+        <p>A: No, this is just an initiative for promoting open-source cultue along with doing a solid project. There's no stipend since this is just to hone your skills.</p>
+        
+        <p>Q: What is the time duration of the project?</p>
+        <p>A: Each project is around two-month long.</p>
+        
+        <p>Q: what are the Starting and ending dates of the program?</p>
+        <p>A: We will announce it very soon , just stay tuned.</p>
+        
+        <p>Q: How many mentors will be there for eaach project?</p>
+        <p>A: This will depend on the number of people who apply for the role of mentor . We will let you know when everything is finalized .</p>
+        
+        <p>Q: FAQs did not answer my question. How can I get connected to the community to ask my question?</p>
+        <p>A: You can get connected to us on <a href="https://t.me/joinchat/AmwdvEAc48xN_P0xRaR_7Q" style="color:red">telegram</a> group for further queries.</a>
```

Comment:
The mailing list would be a far better place for the questions and not the telegram group